src/MLJCatBoostInterface.jl

```diff
@@ -74,9 +75,13 @@ include("mlj_catboostregressor.jl")
 
 const CatBoostModels = Union{CatBoostClassifier,CatBoostRegressor}
 
-function MMI.selectrows(::CatBoostModels, I, X, y)
+function MMI.selectrows(::CatBoostModels, I, X, y, cat_features, text_features)
     py_I = numpy.array(numpy.array(I))
-    return py_X.iloc[py_I,], y[py_I]
+    return X.iloc[py_I,], y[py_I], cat_features, text_features
+end
+
+function MMI.selectrows(::CatBoostModels, I::Colon, X, y, cat_features, text_features)
```

Comment:
I added another version of select rows for when we pass a Colon. Pandas can't handle that as the indexing. I thought that just dispatching 2 versions was a good way to handle this.

Reply:
Good catch!